xgcm/test/test_axis.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from xgcm.axis import Axis
+
+from .datasets import all_datasets, datasets
+
+
+def periodic_1d():
+    # TODO get this from datasets.py
+
+    N = 5
+    ds = xr.Dataset(
+        {"data_g": (["XG"], np.random.rand(N)), "data_c": (["XC"], np.random.rand(N))},
+        coords={
+            "XG": (
+                ["XG"],
+                2 * np.pi / N * np.arange(0, N),
+                {"axis": "X", "c_grid_axis_shift": -0.5},
+            ),
+            "XC": (["XC"], 2 * np.pi / N * (np.arange(0, N) + 0.5), {"axis": "X"}),
+        },
+    )
+    return ds
```

Comment:
Can we just get this from the dictionary in datasets.py?

Reply:
Done